bigtop-packages/src/common/hive/patch12-HIVE-26513-metastore-compatible-hadoop334.diff

```diff
@@ -0,0 +1,13 @@
+diff --git a/pom.xml b/pom.xml
+index cb54806ef5cad6650837e2370207d7272f133c76..6f39c28a601a461fcce937f3c782c44c8e4fc4ba 100644
+--- a/pom.xml
++++ b/pom.xml
+@@ -209,7 +209,7 @@
+     <zookeeper.version>3.4.6</zookeeper.version>
+     <jpam.version>1.1</jpam.version>
+     <felix.version>2.4.0</felix.version>
+-    <curator.version>2.12.0</curator.version>
++    <curator.version>4.2.0</curator.version>
```

Comment:
We are [specifying properties in do-component-build script](https://github.com/apache/bigtop/blob/f92933b35e33db9759dc057058a82d9084fe5e85/bigtop-packages/src/common/hive/do-component-build#L22-L29) to adjust versions of products. If just changing the `curator.version` works, updating the do-component-build should be consistent. @timyuer.

Reply:
I will do it. Thanks. @iwasakims